### PR TITLE
Update UI schema

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
Following changes have been introduced. 
- Update UI schema.
- Update setting two properties.

### CODE REVIEW

File: src/main/resources/uischema.json [hunk 1]
```diff
@@ -139,208 +141,218 @@
...
             "value": {
-              "name": "database.user",
-              "displayName": "Database User",
+              "name": "schema.history.internal.file.filename",
+              "displayName": "Schema History Internal File Filename",
               "inputType": "string",
-              "required": "true",
-              "helpTip": "Name of the database user to use when connecting to the database server."
+              "required": "false",
+              "helpTip": "This value is required only if io.debezium.storage.file.history.FileSchemaHistory was provided for the schema.history.internal value. You need to specify the path to a file where the database schema history is stored. By default, the file will be stored in the <Product home>/cdc/schemaHistory directory."
```
Comment (SanojPunchihewa) [inline src/main/resources/uischema.json]: Shall we use conditional rendering here?

File: src/main/resources/uischema.json [hunk 2]
```diff
@@ -139,208 +141,218 @@
...
             "type": "attribute",
             "value": {
-              "name": "offset.storage.replication.factor",
-              "displayName": "Offset Storage Replication Factor",
+              "name": "connector.name",
+              "displayName": "Connector Name",
               "inputType": "string",
               "required": "false",
-              "helpTip": "Replication factor used when creating the offset storage topic. Required when offset.storage is set to the <…>.KafkaOffsetBackingStore."
+              "helpTip": "Unique name for the connector."
```
Comment (arunans23) [inline src/main/resources/uischema.json]: Shall we explain this a little bit more?

File: src/main/resources/uischema.json (hunk 1 above)
Comment (chathuranga-jayanath-99) [reply src/main/resources/uischema.json]: Sure

File: src/main/resources/uischema.json (hunk 2 above)
Comment (chathuranga-jayanath-99) [reply src/main/resources/uischema.json]: Sure. I'll add.

**Review by SanojPunchihewa — APPROVED** [review]

**Pull request merged**